.github/workflows/image_release.yaml

```diff
@@ -0,0 +1,48 @@
+name: Image Release
+
+on:
+  workflow_dispatch:
```

Comment:
This manual trigger is fine for now, but eventually we will want it so that we have a single entry point for creating a release.

This should create any release commits (update versions etc), tag the release, push to to the repository and then trigger the Operator image and bundle release.